src/promptflow-azure/tests/sdk_cli_azure_test/e2etests/test_telemetry.py

```diff
@@ -145,6 +145,7 @@ def check_evelope():
                         "error_target",
                         "error_message",
                         "error_detail",
+                        "mac_address_hash",
```

Comment:
Why do we need to add this? 

we already have a installation_id field which computed similarily like this.

Reply:
Can you please point me to it ? Is it captured for local runs ?